README.md

```diff
@@ -15,51 +15,92 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(max(l))
 ```
-
 - [Min Value](https://repl.it/@Admin7/minvalue)
+
 ```python
-# copy & paste your solution here
+
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(min(l))
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+def return_extended(l, iter):
+  for i in iter:
+    l.append(i)
+  return l
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
-# copy & paste your solution here
+def create_matrix(h, w):
+  result = []
+  for i in range(h):
+    row = []
+    for x in range(w):
+      row.append('#')
+    result.append(row)
+  return result
 ```
 
 
 #### Dictionaries
 - [Receipt Tally](https://repl.it/@Admin7/receipttally)
 ```python
-# copy & paste your solution here
+def get_total(h, w):
+  total = 0
+  new = 0
+  for i in h:
+    for key,value in i.items():
+        if value == w:
+            total = i.get('subtotal')
+  return total
 ```
 
 - [Translation](https://repl.it/@Admin7/translations)
 ```python
-# copy & paste your solution here
+def translation(word, lang):
+  ans = ''
+  for key,value in translations.items():
+    if key == word:
+      ans = translations[key]
```

Comment:
I wouldn't assign this to ans for two reasons:
1. It's not descriptive. A dictionary of translations is not the result you're looking for!
2. What if something went wrong and you returned ans as a dictionary? Not only is that not what you were expecting -- it's also a different type! It could break your code down the line if this result was passed into another function or server to a user.

Don't be afraid to create new variables. `transalationsForLang` would work!